www/docs/en/dev/config_ref/images.md

```diff
@@ -61,17 +57,16 @@ Android
                   <icon src="res/android/xhdpi.png" density="xhdpi" />
          </platform>
 
-BlackBerry10
+##BlackBerry10
```

Comment:
Nitpick: for consistency with other headings, please add a space after the `#`s.